README.md

```diff
@@ -130,7 +130,7 @@ An overview of the project folders can be seen below.
 Code references are provided for all platform building blocks. That includes the base library, the VM, a command line application and the compiler. 
 
 * [neo:](https://github.com/neo-project/neo/) Neo core library, contains base classes, including ledger, p2p and IO modules.
-* [neo-modules:](https://github.com/neo-project/neo-modules/) Neo modules include additional tools and plugins to be used with Neo.
+* [neo-express:](https://github.com/neo-project/neo-express/) A private net optimized for development scenarios.
 * [neo-devpack-dotnet:](https://github.com/neo-project/neo-devpack-dotnet/) These are the official tools used to convert a C# smart-contract into a *neo executable file*.
```

Comment:
Also, let's add proposals repo and non-native-contracts repo. These two should also be mentioned here, they are an important part of the project.

Reply:
agrred